snoop/data/models.py

```diff
@@ -78,6 +78,8 @@ class Blob(models.Model):
     def __str__(self):
         return self.pk
 
+    __repr__ = __str__
+
```

Comment:
I'm pretty sure this should not be necessary, the default `__repr__` is sane ([on master](https://github.com/django/django/blob/5a4d7285bd10bd40d9f7e574a7c421eb21094858/django/db/models/base.py#L518-L519); also [in version 2.2.8](https://github.com/django/django/blob/2.2.8/django/db/models/base.py#L518-L519) that we currently use)